.golangci.yaml

```
@@ -27,6 +27,8 @@ linters:
     #- ireturn
     - nilnil
     - nilerr
+    #- goconst
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)